.github/pull_request_template.md

```diff
@@ -15,6 +15,7 @@ If **yes**, describe the new output and/or changes to the existing output:</br>
 
 ## Pull Request Checklist ##
 See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
+- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
```

Comment:
I don't know that this is a step that is needed as part of creating a pull request. This should have been done for completing the issue, so it should be covered there. Again, it doesn't hurt to include it.